src/browser/components/molecules/TemplateTaskForm/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import InputField from '../../atoms/InputField'
+import Button from '../../atoms/Button'
+
+export default class TemplateTaskForm extends Component{
+
+  render() {
+    return(
+      <div>
+        <form onSubmit={(event)=>this.props.submit(event)}>
+          <label>
+            <InputField label='name' name='name' update={(event, inputField)=>this.props.update(event, inputField)} />
```

Comment:
I am a fan of using the `bind` function, as it looks less verbose. So it would change to 
```xml
 <InputField label='name' name='name' update=this.props.update.bind(this)} />
```